tests/python/unittest/test_gluon.py

```diff
@@ -665,6 +665,34 @@ def transpose(shape):
         assert (layer(x).shape==ceil_out_shape)
 
 
+@with_seed()
+@pytest.mark.parametrize('cudnn_off', [True, False])
+@pytest.mark.parametrize('variable', ['running_var', 'running_mean'])
+def test_batchnorm_backward_synchronization(cudnn_off, variable):
+    """
+    Tests if synchronization of BatchNorm running variables is done correctly.
+    If not, the test sometimes fails - depending on the timing.
+    """
+    ctx = mx.cpu() if cudnn_off else mx.gpu()
```

Comment:
It should be `ctx=default_context()`.

In CI, there are tests on CPU and GPU. We don't need to specify the context.

Reply:
Thanks for the tip @wkcn,  
Is there any command line option or environment value to enforce particular context  when I run pytest locally?

Reply:
There seems to no command line to specify the context.
The default context is CPU.
We can create a python file, like https://github.com/apache/incubator-mxnet/blob/master/tests/python/gpu/test_gluon_gpu.py

'''
import mxnet ad mx
from some_test_file import *
set_default_context(mx.gpu(0))
'''
It will switch to gpu context.